### ISSUE TITLE
Another Communication / Behavior score bug issue

### ISSUE THREAD

**Issue by ZhiHuiC** [ValveSoftware/Dota2-Gameplay#14574]
### Description

Hi Valve,

I've been consistently dropping both behavior and communication score, despite barely communicating, only with the occasional banters in every or so game.
My last few summaries I've had an average of 5 or so reports (most of them are actually ridiculous since people flame, blame and then report you, or just simply report you because you're not playing up to their standards.
My last match as you can see, I've been reported 3 times and lost 400 communication score, when I barely even type chat in game.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/15137041/4ef47d14-e6d1-4e2e-9b44-3e7cd2019fd2)

This doesn't really make sense, since in the past as you can see below, I've been reported consistently on average about 10 times, but still managed to keep max score up to 12000, going up and down.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/15137041/b623db94-5f00-4d71-b761-30dcfd82d7d8)

I haven't abandoned in over 210 games, although I have abandoned 3x since my last 1000 games.
I did notice that the score consistently started dropping since the abandons, I'm not sure if this is what triggered it, but it feels impossible to get back up despite keeping chat and disruption to a minimal.

I play mainly turbo games.
Is this a known bug/issue? 
Right now it just feels like I'm being punished for actually just playing the game and barely saying anything, whereas previously I've received much more reports but somehow managed to stay in 12k.

For my second screenshot, it doesn't make sense that I've  been reported 9x , and received over 200 increased points, whereas 3 reports now will cost me, and I lose ~400 communication score?

### Example Match ID (and possibly Timestamp)

7498798028

**Comment by ZhiHuiC** [comment]
also to add to this, my latest communication score is 7594. being down from just under 8k previously. 
So I lost about 400 score from reports? Are there any other conditions that contribute towards the score going down?

**Comment by commentator24** [comment]
i dont know how it currently works, but from what i observed:

commends counter is not increasing on player profiles. broken link?
Excessive reports is not working in october. your profile had 19 reports on october 22 but Excessive reports still No, and you didn't even lose any points from 12000 the following week. does it even decrease score properly? related to broken link to profiles?
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/bc8993ac-2ab6-4480-a794-4dab8e8a59fd)
6 reported matches, total 8 reports and 10 commends, score seems to increase by +107. 50 each? which is odd since you should be losing score.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/207a924f-7455-46ef-90c8-9c8ce2609e3a)
the following week was similar, but only +60. less reports too, which is odd. shouldn't it be the other way around?
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/468e243d-1308-49fb-ad25-7d4cd3312e03)
and this one is weirder. you had only 1 report but gain only +28. the following summary -272 with 4 reports in 2 games, 10 commends. maybe nothing to do with commends count now.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/80754a5c-d903-48f9-9976-e27ebb95ce7d)

in summary:
i think previously it's not decreasing scores properly in october. now it's somewhat working, and doesn't care if you had less or more than 3 reports (1 or 4+ reports count) you did not lose 400 score for every 3 reports. maybe 50-68 score for each report? the number increase oddly like +28. maybe related to duration of the game?
maybe if the previous summary you had more reports, you will gain less score or something, or perhaps it is simply waiting for Overwatch reports on you completed before causing your score to decrease.

**Comment by ZhiHuiC** [comment]
yea to me all the figures doesn't really add up or make any sense, so I'm not exactly sure what it is or what's causing it.
My main concern now is actually my communication score, which is below 8k which I can't use voice. And for some reason, without using voice makes it harder to communicate, 
My communication score seems to be detached from my behavior, and is consistently going down despite me barely saying much, besides the fact that sometimes I like to give instructions or tell team what to do in order to win.

For my last summary, I lost around 400 communication score with 2 communication reports (for whatever reason) and having 9 commends. How does this even work?

Right now it just seems to me that once you've gone below 10k behavior score, or the current system, you're just being punished for whatever you're doing, by playing the game and my score is just consistently going down no matter what happens

**Comment by ZhiHuiC** [comment]
For my latest summary, I've received 37 behavior score for 1 reported in 15 games and 11 commends.
I'm not complaining for the fact that I'm going up, but I just find it really silly if I'm going to have to play over 1000 games and receiving only 30 points for only 1 report, and losing within the hundreds for other reasons, since with the nature of the game, I  (and others) will always get reported from other party's if someone simply doesn't like you or you're just not playing how they like.
This isn't consistent and a good way to increase the score, it's actually impossible.
